pipeline_api/src/test/java/com/splicemachine/pipeline/PipelineTest.java

```diff
@@ -50,6 +51,7 @@
  * @author Scott Fines
  *         Date: 12/23/15
  */
+@Ignore // doesn't work (No PipelineTestDataEnv found)
```

Comment:
This is supposed to run using the classpath of `hbase_pipeline` or `mem_pipelin` (it's an architecture specific test).

I just noticed this tests are not running on jenkins or as part of the normal maven tests, so we need to fix that.

Reply:
ok, i removed this commit